recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -109,43 +135,42 @@ def upload_package(feedstock_root, recipe_root, config_file):
     specific_config = safe_load(open(config_file))
     if "channel_targets" in specific_config:
         channels = [c.strip().split(" ") for c in specific_config["channel_targets"]]
-        source_channels = ",".join([c.strip() for c in specific_config["channel_sources"]])
+        source_channels = ",".join(
+            [c.strip() for c in specific_config["channel_sources"]])
     else:
         update_global_config(feedstock_root)
         channels = _global_config["channels"]["targets"]
         source_channels = ",".join(_global_config["channels"]["sources"])
 
     if "UPLOAD_ON_BRANCH" in os.environ:
         if "GIT_BRANCH" not in os.environ:
-            print("WARNING: UPLOAD_ON_BRANCH env variable set, but GIT_BRANCH not set. Skipping check")
+            print(
+                "WARNING: UPLOAD_ON_BRANCH env variable set, "
+                "but GIT_BRANCH not set. Skipping check")
         else:
             if os.environ["UPLOAD_ON_BRANCH"] != os.environ["GIT_BRANCH"]:
-                print("The branch {} is not configured to be uploaded".format(os.environ["GIT_BRANCH"]))
+                print(
+                    "The branch {} is not configured to be "
+                    "uploaded".format(os.environ["GIT_BRANCH"]))
                 return
 
-    upload_to_conda_forge = any(owner == "conda-forge" for owner, _ in channels)
+    upload_to_conda_forge = any(owner == TARGET_OWNER for owner, _ in channels)
     if upload_to_conda_forge and "channel_sources" in specific_config:
-        unknown_channel = False
-        allowed_channels = ["conda-forge", "conda-forge/label/", "defaults", "c4aarch64", "c4armv7l"]
+        allowed_channels = [
+            "conda-forge", "conda-forge/label/", "defaults", "c4aarch64",
+            "c4armv7l"]
         for source_channel in source_channels.split(","):
             for c in allowed_channels:
                 if source_channel.startswith(c):
                     break
             else:
-                print("Uploading to conda-forge with source channel '{}' is not allowed".format(source_channel))
+                print(
+                    "Uploading to %s with source channel '%s' "
+                    "is not allowed" % (TARGET_OWNER, source_channel))
                 return
 
-    upload_script_path = os.path.join(recipe_root, 'conda_forge_ci_setup', 'upload_or_check_non_existence.py')
-    if not os.path.exists(upload_script_path):
-        upload_script_path = os.path.join(os.path.dirname(__file__), 'upload_or_check_non_existence.py')
-
-    import importlib.util
-    spec = importlib.util.spec_from_file_location('upload_or_check_non_existence', upload_script_path)
-    mod = importlib.util.module_from_spec(spec)
-    spec.loader.exec_module(mod)
-
     for owner, channel in channels:
-        mod.retry_upload_or_check(recipe_root, owner, channel, [config_file])
+        retry_upload_or_check(recipe_root, owner, channel, [config_file])
```

Comment:
`owner` here is still `conda-forge` right? not `cf-staging`

Reply:
I don't think so. We have to upload to cf-staging. This is the owner. It gets set dynamically depending on the `conda_forge_output_validation` parameter in the `conda-forge.yml`.

Reply:
That's not going to do anything. Target channel is set by `conda_build_config.yml` and rendered into ..ci_support files. For eg: https://github.com/conda-forge/numpy-feedstock/blob/master/.ci_support/linux_numpy1.14python3.6.____cpythonpython_implcpythontarget_platformlinux-64.yaml#L7-L8

Reply:
Frack. Thanks. I’ll fix